### ISSUE TITLE
Bookmarks sidebar: "Open All in Tabs" not available on single-item folders

### ISSUE THREAD

**Issue by vlakoff** [MoonchildProductions/UXP#1339]
In my Bookmarks sidebar, I'm very used to right-click on folders and do "Open All in Tabs".

I just discovered that if the folder contains only one bookmark, the command is greyed out…

For convenience and [consistency](https://en.wikipedia.org/wiki/Principle_of_least_astonishment), I think the command should be available exactly the same.

**Comment by wolfbeast** [comment]
I disagree. An active "open all" command does not make sense if there are no items or there is only one item to open. Pulling in POLA is really not relevant as there are no unexpected behaviors resulting from user action here.

**Comment by vlakoff** [comment]
I'm very used to this command "Open All in Tabs". That's how I open my bookmarks almost every time (and that happens a lot), after I have grouped them into folders.

So, I use the command by "muscle memory". But when the command is disabled, it completely breaks my workflow, it stops me and I have to think "ah, yes, here I can't use it", then open the folder, then middle-click on the bookmark…

**Comment by vlakoff** [comment]
Actually, most often I don't do "right click > Open All in Tabs"… I directly middle-click on the folder.

**Comment by wolfbeast** [comment]
Since middle-clicking means "Open the selected item(s) in a new tab/new tabs", it would make sense for the shortcut to extend to folders where they only have one item.

This would probably mean we have to split up the logic to make this happen depending on the caller.

**Comment by wolfbeast** [comment]
This needs to be moved to the Pale Moon repo since it's front-end work.
